lsp/termux_language_server.lua

```diff
@@ -0,0 +1,10 @@
+---@brief
+---
+--- https://github.com/termux/termux-language-server
+---
+--- Language server for some specific bash scripts.
```

Comment:
? need a more meaningful description. presumably this is for termux users? or is it a general PKGBUILD ls?
```suggestion
--- Language server for various bash scripts such as PKGBUILD, gentoo ebuild, termux <something>, etc.
```

Reply:
Hi, Termux package maintainer here.
The appropriate term would be Termux build.sh
```suggestion
--- Language server for various bash scripts such as PKGBUILD, Gentoo ebuild, Termux build.sh, etc.
```

Reply:
> I was unsure how to formulate that, since `termux_language_server` does a a lot of things.
> 
> How about:
> 
> > Language server for Termux build scripts, Arch/Gentoo PKGBUILDs, and other distro-specific Bash packaging files.

That feels a bit imprecise.
To me that would imply that Arch and Gentoo share a build script format, which they do not.

Something like;
`Language server for writing build script files for Arch, Gentoo and Termux packages.`
would be fine, I think the important thing is keeping it clear that they're distinct formats.

> This may get outdated as soon as new features are added ofc. Your suggestion would also be fine with me.

I'm not sure to what extent @FreedWu plans to extend the scope of this LSP project.